geode-assembly/src/acceptanceTest/java/org/apache/geode/client/sni/ClientSNICQAcceptanceTest.java

```diff
@@ -22,6 +22,7 @@
 import static org.apache.geode.distributed.ConfigurationProperties.SSL_REQUIRE_AUTHENTICATION;
 import static org.apache.geode.distributed.ConfigurationProperties.SSL_TRUSTSTORE;
 import static org.apache.geode.distributed.ConfigurationProperties.SSL_TRUSTSTORE_PASSWORD;
+import static org.apache.geode.test.awaitility.GeodeAwaitility.await;
```

Comment:
Now that we have `NotOnWindowsDockerRule`, we have the ability for a test to start containers once and to break actual testing into multiple test methods. We do it that way (in `SingleServerSNIAcceptanceTest` and `DualServerSNIAcceptanceTest`) for performance reasons—we don't want to take too many minutes out of our CI pipeline, for SNI testing.

So for these reasons I think we should do one of two things:

1. use `NotOnWindowsDockerRule` here, and break these tests into multiple test methods
2. go further, and transfer these CQ tests over to the `SingleServerSNIAcceptanceTest` class (eliminating this test class entirely)

I can see why we might want to do (1) even though it will add more Docker startup/teardown time (versus (2)), since there is some CQ-specific stuff here: 3 fields and an inner (`CqListener`) class.

Reply:
There's only one test in this class.  No need for the ClassRule, though we could standardize on using it if we want.  The SingleServer test expects the same content to be in the server cache for each test, so I don't think it would be appropriate to add this test to it without addressing that.  I would rather keep this class and have it focus on subscriptions.

Reply:
I've changed this to a ClassRule